### DISCUSSION TITLE
what FORMAT please

### DISCUSSION THREAD

**Discussion by Elektraglide** [keirf/greaseweazle#298] [Q&A] [answered]
disk-analyse on an .IMD for a Tektronix 4404 says:
IBM-MFM DD (8 512-byte sectors, 4096 bytes)

Which format does that map to for GW?

This looks about right?
```
disk pc98.2d
    cyls = 40
    heads = 2
    tracks * ibm.mfm
        secs = 8
        bps = 512
        gap3 = 57
        rate = 250
        rpm = 300
    end
end
```

Adam

**Comment by keirf** [comment]
Looks likely. Does it work?

**Comment by Elektraglide** [comment]
yes thx.  though I'm not using the gap3.

QQ: My drive reports 299rpm.  Should I be using --fake-index=299rpm to account for that?

Also drive keeps randomly failing to raise track00 on Cylinder 0, and sometimes does raise track00 on Cylinder != 0..
The photo-detector looks clean enough but wondering whether the NAND gates on the way to the ST506 are dying after 40 years..

**Reply by keirf** [reply, marked as answer]
Don't use `--fake-index`, it's mainly for reading flippy single-sided disks which appear to have no index sensor hole when flipped. The 299rpm is not a problem. Any later analysis can account for this unaided.

It would be unusual for the solid-state trk0 sense circuit to go bad. Suspect the moving parts instead, eg. sticky worm gear.
